### PR TITLE
Migrate TravisCI jobs to Github Actions

### PR DESCRIPTION
**Pull request by omeryahud**
**What this PR does / why we need it**:
Migrate TravisCI jobs to GithubActions because of TravisCI's rate limitations

On Release:
- Build the common-templates bundle
- Publish bundle as a release asset
- Update the bundle in the ssp-operator repo by opening a PR

**Special notes for your reviewer**:
Tested on my fork:
- Release that triggered the job (the assets were populated by the triggered action): https://github.com/omeryahud/common-templates/releases/tag/v0.0.32
- Triggered action: https://github.com/omeryahud/common-templates/runs/1925516852?check_suite_focus=true

```release-note
None
```

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        with:
+          upload_url: ${{ steps.get_release.outputs.upload_url }}
+          asset_path: ./dist/common-templates.yaml
+          asset_name: common-templates.yaml
+          asset_content_type: text/plain
+
+      - name: Update SSP Operator
+        run: |
+          # Install required packages
+          curl -L https://golang.org/dl/go1.16.linux-amd64.tar.gz -o go.tar.gz
```
Comment (akrejcir) [inline .github/workflows/release.yaml:45]: Maybe you can use `actions/setup-go` to install golang. https://github.com/marketplace/actions/setup-go-environment

**Review by akrejcir — COMMENTED** [review]
Looks good. I have just one suggestion.

File: .github/workflows/release.yaml (hunk 1 above)
Comment (omeryahud) [reply .github/workflows/release.yaml:45]: Yes, that's a great idea! I will update the code

**Pull request merged**